src/components/Surface/Surface.js

```diff
@@ -527,6 +562,24 @@ function Surface() {
                 payload: { id: argument.id, argumentText },
               });
             }}
+            startEditing={() => {
+              if (argument.id === null) {
+                return;
+              }
+              return dispatch({
+                type: 'setEditMode',
+                payload: { id: argument.id, editMode: true },
+              });
+            }}
+            stopEditing={() => {
+              if (argument.id === null) {
+                return;
+              }
+              return dispatch({
+                type: 'setEditMode',
+                payload: { id: argument.id, editMode: false },
```

Comment:
Should the id be set to null?

Reply:
We use the id to set only the selected argument's editMode to false, but we could in general just set every argument's editMode to false. Since 'setEditMode' is also used when we set the editMode to true, I think we can keep this as it is for now